Userland/Shell/Tests/immediate.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+source $(dirname "$0")/test-commons.inc
+
+# Length
+
+if test "${length foo}" -ne 3 {
+  fail invalid length for bareword
+}
+
+if test "${length "foo"}" -ne 3 {
```

Comment:
> `${ … }` which usually evaluates a single variable

So does ours - `${length $foo}` is more-or-less directly equivalent to `${#foo}`.

Reply:
...wrong place to reply...oops

Reply:
(Since I don't see how to do a "proper" response to the main change request, let's do that here, then.)

Ohh, yeah, now I see how you mean. I was rather thinking about Make, where counting the words of a variable would be considered a function. Sure, then it makes even more sense.

Reply:
I'm not quite sure what you mean by it confusing variables and strings, but I added the `foobar` bareword test case.